src/app/_components/FilterContainer.tsx

```diff
@@ -53,11 +56,13 @@ FilterContainer.MobileFiltersAndResults = function MobileFiltersAndResults({
 
 FilterContainer.DesktopFilters = function DesktopFilters({
   search,
+  location,
   sort,
 }: FilterProps) {
   return (
     <div className="hidden justify-end gap-6 lg:flex">
       {search}
+      <div>{location}</div>
```

Comment:
This is probably where we should set a width, as we do in `MobileFiltersAndResults`, so that the layout doesn't shift as users select options.

Also, why do we check for the existence of `location` above and not here?

```ts
{location && ...}
```

Reply:
So we avoid this in blog layout

![Screenshot 2024-11-25 at 21 48 43](https://github.com/user-attachments/assets/662680a4-019c-4215-b3ea-f95823c29b36)